source/_patterns/20-molecules/teaser/teaser.scss

```diff
@@ -784,6 +787,12 @@
       background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6));
     }
   }
+  .teaser--longform__links {
```

Comment:
Der Linter in mir würde sagen dass da eine leere Zeile vorne dran fehlt, aber es ist Montag und der Linter schläft noch.